stonesoup/models/base.py

```diff
@@ -104,6 +104,32 @@ def function(self, state_vector, noise=None, **kwargs):
         pass
 
 
+class ReversibleModel(NonLinearModel):
+    """Non-linear model containing sufficient co-ordinate
+    information such that the linear co-ordinate conversions
+    can be calculated from the non-linear counterparts.
+
+    Contains an inverse function which computes the reverse
+    of the relevant linear-to-non-linear function"""
+
+    def inversefunction(self, state_vector, **kwargs):
```

Comment:
This should have an `@abstractmethod` decorator.

Also, for consistency with other methods, this should be `inverse_function`.